android/src/main/java/com/RNFetchBlob/RNFetchBlobReq.java

```diff
@@ -161,7 +161,8 @@ public void run() {
             if (options.addAndroidDownloads.getBoolean("useDownloadManager")) {
                 Uri uri = Uri.parse(url);
                 DownloadManager.Request req = new DownloadManager.Request(uri);
-                if(options.addAndroidDownloads.getBoolean("notification")) {
+                Boolean hasNotification = options.addAndroidDownloads.hasKey("notification") ? options.addAndroidDownloads.getBoolean("notification") : false;
```

Comment:
can simplify logic to
`options.addAndroidDownloads.hasKey("notification") && options.addAndroidDownloads.getBoolean("notification")`